### PR TITLE
Remove the DBus support for anaconda

### PR DESCRIPTION
**Pull request by poncovka**
The DBus support was moved to a new package called `dasbus`, so
remove the code from `anaconda` and use the new DBus library.

**Related:** https://github.com/rhinstaller/dasbus/pull/1

### CODE REVIEW

File: dasbus/client/proxy.py [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+        except KeyError:
+            pass
+
+        return self._create_member(*key)
+
+    def _create_member(self, *key):
+        """Create a member of the DBus object.
+
+        If the member doesn't exist, ask a handler
+        to create it.
```
Comment (jkonecny12) [inline dasbus/client/proxy.py]: This can be one line comment.

File: dasbus/client/proxy.py [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+            return member.get()
+
+        return member
+
+    def __setattr__(self, name, value):
+        """Set the attribute.
+
+        Called when an attribute assignment is attempted.
+        Call the DBus handler if the the name is not a
+        name of an instance attribute defined in _locals.
```
Comment (jkonecny12) [inline dasbus/client/proxy.py]: Please fix line length of doc strings in this file.

**Comment by poncovka** [comment]
Jenkins, test this please.

**Comment by poncovka** [comment]
Rebased.

**Comment by poncovka** [comment]
Jenkins, test this please.

**Pull request merged**